pkg/scheduler/core/equivalence_cache_test.go

```diff
@@ -692,13 +746,15 @@ func TestInvalidateCachedPredicateItemOfAllNodes(t *testing.T) {
 
 	// there should be no cached predicate any more
 	for _, test := range tests {
-		if algorithmCache, exist := ecache.algorithmCache[test.nodeName]; exist {
-			if _, exist := algorithmCache.predicatesCache.Get(testPredicate); exist {
-				t.Errorf("Failed: cached item for predicate key: %v on node: %v should be invalidated",
-					testPredicate, test.nodeName)
-				break
+		t.Run(test.nodeName, func(t *testing.T) {
+			if algorithmCache, exist := ecache.algorithmCache[test.nodeName]; exist {
+				if _, exist := algorithmCache.predicatesCache.Get(testPredicate); exist {
+					t.Errorf("Failed: cached item for predicate key: %v on node: %v should be invalidated",
+						testPredicate, test.nodeName)
+					return
```

Comment:
This won't exit the loop like the break did previously. I think it should be fine to run to the end of the loop though.